packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -75,11 +128,15 @@ export class CustomScrollbar extends Component<Props> {
       passedProps.style.display = 'none';
     }
 
-    return <div {...passedProps} className={track} />;
+    return (
+      <div {...passedProps} className={track === 'track-vertical' ? styles.trackVertical : styles.trackHorizontal} />
```

Comment:
Instead of passing the string argument `track` ('track-vertical' | 'track-horizontal') shouldn't we pass the correct className instead? This would eliminate the ternary in className:
`className={track === 'track-vertical' ? styles.trackVertical : styles.trackHorizontal}`